src/Turing.jl

```diff
@@ -37,22 +37,27 @@ include("stdlib/distributions.jl")
 include("stdlib/RandomMeasures.jl")
 
 """
-struct Model{F, Targs <: NamedTuple}
+struct Model{F, Targs <: NamedTuple, Tmissings <: Val}
```

Comment:
Could you add a line explaining the fields of Model?

Reply:
Sure.